aiohttp/web_reqrep.py

```diff
@@ -321,6 +321,15 @@ def json(self, *, loads=json.loads, loader=None):
         return loads(body)
 
     @asyncio.coroutine
+    def multipart(self, *, reader=multipart.MultipartReader):
+        """Return async iterator to process BODY as multipart."""
+        if not self.content_type.startswith('multipart/'):
+            warnings.warn('Attempt to process multipart request with'
```

Comment:
`application/x-www-form-urlencoded` - is about `key=value&anotherkey=anothervalue` format. It's not multipart thing. You probably mean `multipart/form-data` which is used to transfer binary data like files. 

From the [spec](https://www.w3.org/TR/html401/interact/forms.html):
> This attribute specifies the content type used to submit the form to the server (when the value of method is "post"). The default value for this attribute is "application/x-www-form-urlencoded". The value "multipart/form-data" should be used in combination with the INPUT element, type="file".